src/main/resources/default/taglib/w/modal.html.pasta

```diff
@@ -1,11 +1,12 @@
 <i:arg name="name" type="String" />
 <i:arg name="titleKey" type="String" default="" />
 <i:arg name="title" type="String" default="@i18n(titleKey)" />
+<i:arg name="scrollable" type="boolean" default="false" />
 
 <i:pragma name="description" value="Renders a body for a modal dialog within a Wondergem template" />
 
 <div id="@name" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="@(name + ' Title')">
-    <div class="modal-dialog" role="document">
+    <div class="modal-dialog @if(scrollable){modal-dialog-scrollable}" role="document">
```

Comment:
@modal-dialog-scrollable="scrollable"

Reply:
Resulted in "Unknown variable modal"
`       <div class="modal-dialog @modal-dialog-scrollable='scrollable'" role="document">
`